crates/sovereign-sdk/rollup-interface/src/state_machine/soft_confirmation.rs

```diff
@@ -25,6 +25,19 @@ pub struct UnsignedSoftConfirmation<'txs, Tx> {
     timestamp: u64,
 }
 
+/// TODO: doc or find workaround
+#[derive(BorshSerialize)]
+pub struct OldFormat<'txs> {
```

Comment:
```suggestion
pub struct UnsignedSoftConfirmationV1<'txs> {
```

Reply:
These changes are cherry-picked from #1514 
I believe I should wait for him to change these parts